helpers/manifest-parser.js

```diff
@@ -141,8 +141,13 @@ function parseIcon(raw) {
   let sizes = parseString(raw.sizes);
   if (sizes.value !== undefined) {
     let set = new Set();
-    sizes.value.split(/\s/).forEach(size => set.add(size.toLowerCase()));
-
+    sizes.value.split(/\s/).forEach(size => {
+      const pair = size.split(/x/i);
+      if (pair[0] !== pair[1]) {
```

Comment:
IMO i think the check belongs here in the parser. I'd be in favor of exposing the debugString for any icon-size check audits, for the winning pair. 

e.g. if theres 3 icons: (92x128, 144x144, 192x192), then the user never sees the debugString.

seem good, brendan?